org.gnu.emacs.json

```diff
@@ -17,6 +17,25 @@
         "--filesystem=/var/tmp"
     ],
     "modules": [
+        {
+            "name": "jansson",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://digip.org/jansson/releases/jansson-2.13.1.tar.gz",
+                    "sha256": "f4f377da17b10201a60c1108613e78ee15df6b12016b116b6de42209f47a474f"
+                }
+            ],
+            "cleanup": [
+                "/include",
+                "/bin",
+                "/share",
+                "/lib/pkgconfig",
+                "/lib/*.la",
+                "/app/lib/libjansson.a"
```

Comment:
This probably should be just /lib/libjansson.a - similar to how you remove the other bits.

Reply:
Ouch, yes, sorry.